docs/team/dionegoh.md

```diff
@@ -11,42 +11,55 @@ It is written in Java and has over 10kLoC.
+   * Justification: We added more attributes for a friend, such as description and logs. We also added events which
+      contain references to `FriendName`. Thus, it is essential for the user to be able to view all these new features 
+      for an individual friend in an uncluttered manner.
+   * Highlights: Created an `ExpandedPersonCard` showing the full details of a friend.
 
-* **deletefriend**: Added the functionality to allow user to delete a friend from the address book.
-    * What it does: Allows users to delete a friend from the address book by typing in the relevant command into the GUI. 
-    * Justification:
+2. **New Feature: command aliases**
+   * What it does: Enables users to enter shorter command aliases instead of the usual command.
+   * Justification: Allows experienced users familiar with the commands to enter commands faster.
+   
+3. **Enhancement to existing feature `addfriend`**: 
+    * What it does: Allows users to add a friend into address book by typing in the relevant command into the GUI.
     * Highlights:
-    * Credits: 
+      * Added a new `Description` field for a `Friend`
+      * Made other fields in optional and only `FriendName` compulsory, so that users who do not know full details of a friend
+        can still add the friend into `Amigos`.
+      
+4. **Enhancement to existing feature `deletefriend`**: 
+    * What it does: Allows users to delete a friend from `Amigos`.
+    * Highlights: Users can delete a friend either by entering the `FriendName` or a valid `Index`. 
+      Users can delete an existing friend by `FriendName` even if the
+      friend is not currently being shown on the filtered list on the GUI. 
 
-* **editfriend**: Added the functionality to allow user to edit details of a friend from the address book.
-    * What it does: Allows users to edit a friend from the address book by typing in the relevant command into the GUI.
-    * Justification:
-    * Highlights:
-    * Credits:
+5. **Enhancement to existing feature `editfriend`**:
+    * What it does: Allows users to edit a friend from `Amigos`.
+    * Justification: Users can edit a friend either by entering the `FriendName` or a valid `Index`.
+    * Highlights: The prefix `cn/` is used to identify the current `FriendName`. `nn/` is used for the
+      edited `FriendName`.
+
+6. **Enhancement to existing feature `findfriend`**:
+    * What it does: Allows users to search for friends from `Amigos` using keywords.
+    * Highlights: Keywords are not limited to `FriendName`. `Tag` and `LogName` of a Person are also
+      searched. The user input is split according to the prefixes (`n/` for `FriendName`, `t/` for `Tag`,
+      `ttl` for `LogName`). Also, keywords do not have to be matched fully. As long as a substring is matched, then 
+      the friend would be shown.
 
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s2.github.io/tp-dashboard/?search=dionegoh&breakdown=true)
 
 * **Project management**:
```

Comment:
I think you can also mention being in charge of a specific milestone as well, if you want, and being in charge of certain components.